packages/client/src/index.ts

```diff
@@ -29,6 +29,7 @@ export class LogtoClient {
   public oidcReady = false;
   public issuer: Issuer<Client> | null = null;
   private client: Client | null = null;
+  private tokenSet: TokenSet | null = null;
```

Comment:
would prefer to use `Optional` in `@silverhand/essentials` for consistency of optional stuff. `null` can be used in something that *explicitly* empty, or an explicit setting of `null`.